x-pack/filebeat/input/httpjson/input.go

```diff
@@ -159,6 +161,16 @@ func run(
 	return nil
 }
 
+// The Request.Tracer.Filename may have ":" when a httpjson input has cursor config
```

Comment:
Why is this capitalized? 

`s/Request.Tracer.Filename/request.tracer.filename/`